ldi-orchestrator/ldio-application/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/services/PipelineStartupService.java

```diff
@@ -0,0 +1,34 @@
+package be.vlaanderen.informatievlaanderen.ldes.ldio.services;
+
+import be.vlaanderen.informatievlaanderen.ldes.ldio.services.initializer.ConfigPipelineInitializer;
+import be.vlaanderen.informatievlaanderen.ldes.ldio.services.initializer.FileStoredPipelineInitializer;
+import be.vlaanderen.informatievlaanderen.ldes.ldio.services.initializer.PipelineInitializer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.event.ContextRefreshedEvent;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Service;
+
+import java.util.stream.Stream;
+
+@Service
+public class PipelineStartupService {
+	private final Logger log = LoggerFactory.getLogger(PipelineStartupService.class);
+
+	private final Stream<PipelineInitializer> pipelineInitiators;
+
+	public PipelineStartupService(ConfigPipelineInitializer configPipelineInitializer,
+	                              FileStoredPipelineInitializer fileStoredPipelineInitializer) {
+		this.pipelineInitiators = Stream.of(fileStoredPipelineInitializer, configPipelineInitializer);
+	}
+
+	@EventListener
+	public void initPipelines(ContextRefreshedEvent event) {
```

Comment:
Would this do the same without having an unused param?

```	@EventListener(ContextRefreshedEvent.class)
	public void initPipelines() {```

Reply:
Nope. But i'll rename the even to ignored so that it's a bit more clear